docs/developer-docs/integrations/sns/index.md

```diff
@@ -3,16 +3,17 @@
 ## Technical documentation
 
 Here you will find
-* [an intro to sns](sns-intro.md)
+* [An intro to SNS](sns-intro.md)
 
-deployment info
-* [Launch intro](deployment/launch-intro)
-* [frontend](deployment/frontend-integration)
-* [Predeployment](deployment/predeployment)
-* [Local testing](deployment/local-testing)
-* [deployment](deployment/deployment.md)
+Info how to get an SNS
+* [Intro](get-sns/get-sns-intro)
+* [Frontend integration preparation](get-sns/frontend-integration)
+* [Preparation](get-sns/preparation)
+* [Local testing](get-sns/local-testing)
+* [Getting an SNS in production](get-sns/get-sns-production)
```

Comment:
all links have to have the extensions included. Otherwise docusaurus will screw everything up. Please also adjust the following 6 lines.

Reply:
Could you elaborate on what you mean by "extensions included"?
For a concrete example, do you mean that I have to change
[Intro](get-sns/get-sns-intro)
to 
[Intro](get-sns/get-sns-intro.md)
?

Reply:
done in next commit (also in other lines in this file)